string edges          # json { node: [nodes] }
string values         # json { node: Datapoint }
string timestamps     # json { "(node, node)": [timestamp] }
string labels         # json { "(node, node)": label }
string initialState   # initial node
